lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -44,6 +55,7 @@
   - When using ca_cert on OS X it has been reported that in some conditions the validate will always succeed.
 
 requirements:
+  - "python >= 2.7"
```

Comment:
```suggestion
  - "python >= 2.7 when using C(proxy_host)"
```

Reply:
Thank you for pointing it out!
I will modify requirements.